crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,62 @@
+from collections import namedtuple
+from enum import Enum
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+ConfigEntry = namedtuple("ConfigEntry", ["config_name", "default_value"])
+
+
+class NoteSortingMethods(Enum):
+    FIELD1 = "field1"
+    FIELD2 = "field2"
+    NO_SORTING = "none"
+    GUID = "guid"
+    FLAG = "flag"
+    TAG = "tag"
+    NOTE_MODEL = "notemodel"
+    NOTE_MODEL_ID = "notemodelid"
+
+
+class ConfigSettings:
```

Comment:
Adding comment to track tests for this one, as I'm getting lost among all outdated ones 😅

Reply:
Testing added :wink: